canopy/canopyjs/src/index.ts

```diff
@@ -17,6 +17,7 @@ export {
   TransactionBuilder,
   SabreTransactionBuilder
 } from './transactions/transaction';
+export { BatchBuilder } from './transactions/batch';
```

Comment:
Spacing looks slightly off here.

How come these aren't exported in the canopy object?